execution/inc/leatherman/execution/execution.hpp

```diff
@@ -430,6 +434,7 @@ namespace leatherman { namespace execution {
      */
     std::tuple<std::string, std::string> process_streams(
             bool trim,
+            bool strip_cr,
```

Comment:
Yeah, I think we should call it out as a change but not worry about planning for the change (as no one should be impacted by it). We could throw it in an anonymous namespace to make it hard to use.